_studio/hevce_hw/h265/include/mfx_h265_encode_plugin_hw.h

```diff
@@ -163,6 +163,12 @@ class Plugin : public MFXEncoderPlugin
     virtual ~Plugin()
     {}
 
+    static Plugin* GetInstance()
+    {
+        static Plugin instance(false);
+        return &instance;
```

Comment:
In previous design each Create call returned new object, currently it return same(!) object for everyone who wants hevc encoder, i.e. they will share all their resources. Is it ok?

Reply:
Yes, this object uses just for check exist of required methods